apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -57,11 +80,21 @@ export default function ListBoxSearch({ selections, model, keyboard, dense = fal
   };
 
   const handleFocus = () => {
+    if (wildCardSearch) {
+      setValue(WILDCARD);
+      setWildcardOn(true);
+    }
     if (!selections.isModal()) {
       selections.begin(['/qListObjectDef']);
     }
   };
 
+  const onBlur = () => {
+    if (wildCardSearch) {
+      setWildcardOn(false);
+    }
+  };
```

Comment:
Remove this, and move into the effect above. `wildcardOn` will be false next time the input is focused since it will trigger a rerender.